src/VirtoCommerce.EventBusModule.Data/VirtoCommerce.EventBusModule.Data.csproj

```diff
@@ -11,7 +11,7 @@
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Azure.Messaging.EventGrid" Version="4.21.0" />
+    <PackageReference Include="Azure.Messaging.EventGrid" Version="4.7.0" />
```

Comment:
What does this downgrade mean? Is it right?

Reply:
Azure.Messaging.EventGrid 4.7.0 is included in the platform.